src/plugin/cesium/cesium.js

```diff
@@ -39,6 +39,7 @@ plugin.cesium.GeometryInstanceId = {
  * @enum {string}
  */
 plugin.cesium.SettingsKey = {
+  ACCESS_TOKEN: 'cesium.accessTokens',
```

Comment:
Given the suggestion to use a single access token, please update this value to `cesium.accessToken`.